doc/HotWordBoosting-Examples.rst

```diff
@@ -0,0 +1,55 @@
+Hot-word boosting API Usage example
+===================================
+
+With DeepSpeech 0.9 release a new API feature was introduced that allows boosting probability from the scorer of given words. It is exposed in all bindings (C, Python, JS, Java and .Net). 
+
+Currently, it provides three methods for the Model class:
+
+- ``AddHotWord(word, boost)``
+- ``EraseHotWord(word)`` 
+- ``ClearHotWords()``
+
+Exact API binding for the language you are using can be found in API Reference.
+
+Example 
+-------
+To use hot-word boosting just add hot-words of your choice before performing an inference to a ``Model``. You can also erase boosting of a chosen word or clear it for all hot-words.
+
+.. code-block:: python
+
+	ds = Model(args.model)
+	...
+	ds.addHotWord(word, boosting)
+	...
+	print(ds.stt(audio))
+	
+Adding boost value to a word many times or erasing hot-word without previously boosting it results in an error.
+
+General usage
+-------------
+
+It is worth noting that boosting non-existent words in scorer (mostly proper nouns) or a word that share no phonetic prefix with other word in the input audio don't change the final transcription. Additionally, hot-word that has a space will not be taken into consideration, meaning that combination of words can not be boosted and each word must be added as hot-word separately. 
+
+
+Positive value boosting
+-----------------------
+
+By adding a positive boost value to one of the words it is possible to increase the probability of the word occurence. This is particularly useful for detecting speech that is expected by the system. 
+
+In the output, large boost value may cause a word following the boosted hot-word to be split into separate letters. This problem is related to the scorer structure and currently only way to avoid it is to adjust boost to a lower value.  
+
+Negative value boosting
```

Comment:
Same, I think there is value that this should be documented within `addHotWord` and its bindings